compiler/arm/codegen/ControlFlowEvaluator.cpp

```diff
@@ -1811,9 +1811,8 @@ static TR::Register *generateMaxMin(TR::Node *node, TR::CodeGenerator *cg, bool
       }
 
    TR_ARMConditionCode cond = max ? (isUnsigned ? ARMConditionCodeHI : ARMConditionCodeGT) : (isUnsigned ? ARMConditionCodeLS : ARMConditionCodeLT);
-   int n = node->getNumChildren();
    int i = 1;
-   for ( ; i < n; i++)
```

Comment:
if you are going to keep the loop just use the getNumChildren. If you aren't going to keep the loop you need to duplicate the code for first and second child. There is no reason to hardcode 2 here since the node / opcode APIs should ensure the number of children is correct before you get here IMO.

Same goes for other codegens.